Makefile

```
@@ -1,9 +1,8 @@
 SHELL=/usr/bin/env bash
-export ACT_PATH:=$(shell pwd)/src:$(ACT_PATH)
 
 all: runtest
 
-truth:
+regression:
 	cd test && ./testgen.sh
```

```suggestion
	cd test && ./generate_regression_truth.sh
```
maybe also rename the generation file to something like this?

---

Good idea